UI/src/view/linear_asset/laneModellingLayer.js

```diff
@@ -11,6 +11,9 @@
     LinearAssetLayer.call(this, params);
     var me = this;
 
+    this.readOnlyLayer = new ViewOnlyLaneModellingLayer(params, map);
```

Comment:
I recommend putting this in assetTypeConfiguration as the traffic sign read only layer is

Reply:
Done